Lib/fontbakery/profiles/shared_conditions.py

```diff
@@ -159,7 +159,7 @@ def glyph_metrics_stats(ttFont):
     ascii_glyph_names = [ttFont.getBestCmap()[c] for c in range(32, 128)
                          if c in ttFont.getBestCmap()]
     ascii_widths = [adv for name, (adv, lsb) in glyph_metrics.items()
-                    if name in ascii_glyph_names]
+                    if name in ascii_glyph_names and adv != 0]
```

Comment:
I'm not sure that this influences the issue report if I understand the change correctly Felipe.  This looks like it is a "is it a monospaced font" side definition change.  The fonts that I tested were designed as monospaced fonts.  We successfully passed that test and the result was correct.  The issue that I found was that *when it is a monospaced font*, there are glyphs outside of the ASCII set that are detected as inappropriately defined in a monospaced font b/c they have zero advance widths.  These were in the combining mark glyph set which are compiled with zero advance widths.   Please let me know if I can provide any additional information